spring-data-jdbc/src/main/java/org/springframework/data/jdbc/repository/support/JdbcQueryLookupStrategy.java

```diff
@@ -325,57 +278,15 @@ NamedParameterJdbcOperations getOperations() {
 		return operations;
 	}
 
-	@SuppressWarnings("unchecked")
-	RowMapper<Object> createMapper(Class<?> returnedObjectType) {
+    QueryMappingConfiguration getQueryMappingConfiguration() {
+        return queryMappingConfiguration;
+    }
 
-		RelationalPersistentEntity<?> persistentEntity = getMappingContext().getPersistentEntity(returnedObjectType);
+    EntityCallbacks getCallbacks() {
+        return callbacks;
+    }
 
-		if (persistentEntity == null) {
-			return (RowMapper<Object>) SingleColumnRowMapper.newInstance(returnedObjectType,
-					converter.getConversionService());
-		}
-
-		return (RowMapper<Object>) determineDefaultMapper(returnedObjectType);
-	}
-
-	private RowMapper<?> determineDefaultMapper(Class<?> returnedObjectType) {
-
-		RowMapper<?> configuredQueryMapper = queryMappingConfiguration.getRowMapper(returnedObjectType);
-
-		if (configuredQueryMapper != null)
-			return configuredQueryMapper;
-
-		EntityRowMapper<?> defaultEntityRowMapper = new EntityRowMapper<>( //
-				getMappingContext().getRequiredPersistentEntity(returnedObjectType), //
-				converter //
-		);
-
-		return new PostProcessingRowMapper<>(defaultEntityRowMapper);
-	}
-
-	class PostProcessingRowMapper<T> implements RowMapper<T> {
```

Comment:
This class is removed completely, since this API was not public in the first place.